src/components/pages/rules/flyouts/ruleNew/ruleNew.js

```diff
@@ -53,69 +76,144 @@ export class RuleNew extends LinkedComponent {
   }
 
   render() {
-    const { onClose } = this.props;
-
+    const { onClose, t } = this.props;
+    const name = this.ruleNameLink.forkTo('name')
+        .withValidator(ruleNameValidator);
```

Comment:
nit: extra indent. I think we do 2 spaces

Reply:
fixed